src/main/java/frc/team2412/robot/sensors/AprilTagsProcessor.java

```diff
@@ -136,7 +136,7 @@ public AprilTagsProcessor(DrivebaseWrapper aprilTagsHelper) {
 		photonCamera = new PhotonCamera(Hardware.PHOTON_CAM);
 		photonPoseEstimator =
 				new PhotonPoseEstimator(
-						fieldLayout, PoseStrategy.MULTI_TAG_PNP_ON_COPROCESSOR, photonCamera, ROBOT_TO_CAM);
+						fieldLayout, PoseStrategy.MULTI_TAG_PNP_ON_RIO, photonCamera, ROBOT_TO_CAM);
```

Comment:
Did you try the coprocessor strategy with the UI option selected? https://github.com/PhotonVision/photonvision/blob/7cec1413410c8053dafd8ecb57095f47469c758c/photon-lib/py/photonlibpy/photonPoseEstimator.py#L36

Reply:
No- The reason I don't think `MULTITAG_PNP_ON_COPROCESSOR` won't work all the time is that since our filters are applied on the Rio side, the result would need to be recalculated on the Rio if the filter changes anything (and thus invalidates the multitag result from the coprocessor).
Ideally we could detect when our filter does anything so that we only calculate on Rio when we have to, but that might take too long to implement for week 1. If the multitag processing takes too long on the Rio, another option is to simply ignore the entire result if anything's bad about it.